.travis.yml

```diff
@@ -26,3 +47,5 @@ script:
   - make
   - if [ $USE_CMAKE == "YES" ] ; then CTEST_OUTPUT_ON_FAILURE=1 make test ; fi
   - if [ $USE_CMAKE == "NO" ] ; then make check ; fi
+  - cppcheck --language=c --std=c89 --verbose --quiet --error-exitcode=1 --enable=warning,performance,portability src/
```

Comment:
As an alternative, we could add using cppcheck as one of the checks in the [Jenkins-hosted tests](https://check.ci.cloudbees.com/job/github-merge-builder/). I've uses the Travis-CI to date as a way to test OSX. I've no problem expanding it or moving extra checks to Jenkins.